Frontend/src/components/Collection.jsx

```
@@ -20,28 +21,33 @@ export default function Collection() {
-              <div key={item._id}>
-                <img
-                  src={item.image}
-                  alt={item.name}
-                  className="w-full h-auto object-cover rounded bg-gray-200"
-                />
-                <h3 className="text-sm font-medium mt-2">{item.name}</h3>
-                <p className="text-sm font-semibold text-gray-800">
-                  ₹{item.price} <span className="text-gray-500">| {item.category}</span>
-                </p>
-              </div>
+              <Link to={`/product/${item._id}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)